specs/_template/03_external_communication.rst

```diff
@@ -0,0 +1,87 @@
+=======  ==================  ======  ==========
+ epic          title         author   created
+=======  ==================  ======  ==========
+`EPIC`_  SPECIFICATION_NAME  AUTHOR  CREATED_AT
+=======  ==================  ======  ==========
+
+.. _EPIC: gnosis/safe#EPIC
+
+=========================
```

Comment:
instead of focusing on external communications here I would rather try to have technical details/ notes here.

This can include external communication, but can also include other technical details (security. format or even platform related)